mobile/src/main/java/de/duenndns/ssl/MemorizingTrustManager.java

```diff
@@ -59,363 +55,350 @@
-			if (isServer)
-				appTrustManager.checkServerTrusted(chain, authType);
-			else
-				appTrustManager.checkClientTrusted(chain, authType);
-		} catch (CertificateException ae) {
-			// if the cert is stored in our appTrustManager, we ignore expiredness
-			ae.printStackTrace();
-			if (isExpiredException(ae)) {
-				Log.i(TAG, "checkCertTrusted: accepting expired certificate from keystore");
-				return;
-			}
-			if (isCertKnown(chain[0])) {
-				Log.i(TAG, "checkCertTrusted: accepting cert already stored in keystore");
-				return;
-			}
-			try {
-				Log.d(TAG, "checkCertTrusted: trying defaultTrustManager");
-				if (isServer)
-					defaultTrustManager.checkServerTrusted(chain, authType);
-				else
-					defaultTrustManager.checkClientTrusted(chain, authType);
-			} catch (CertificateException e) {
-				e.printStackTrace();
-				interact(chain, authType, e);
-			}
-		}
-	}
-
-	public void checkClientTrusted(X509Certificate[] chain, String authType)
-		throws CertificateException
-	{
-		checkCertTrusted(chain, authType, false);
-	}
-
-	public void checkServerTrusted(X509Certificate[] chain, String authType)
-		throws CertificateException
-	{
-		checkCertTrusted(chain, authType, true);
-	}
-
-	public X509Certificate[] getAcceptedIssuers()
-	{
-		Log.d(TAG, "getAcceptedIssuers()");
-		return defaultTrustManager.getAcceptedIssuers();
-	}
-
-	private int createDecisionId(MTMDecision d) {
-		int myId;
-		synchronized(openDecisions) {
-			myId = decisionId;
-			openDecisions.put(myId, d);
-			decisionId += 1;
-		}
-		return myId;
-	}
-
-	private static String hexString(byte[] data) {
-		StringBuffer si = new StringBuffer();
-		for (int i = 0; i < data.length; i++) {
-			si.append(String.format("%02x", data[i]));
-			if (i < data.length - 1)
-				si.append(":");
-		}
-		return si.toString();
-	}
-
-	private static String certHash(final X509Certificate cert, String digest) {
-		try {
-			MessageDigest md = MessageDigest.getInstance(digest);
-			md.update(cert.getEncoded());
-			return hexString(md.digest());
-		} catch (CertificateEncodingException e) {
-			return e.getMessage();
-		} catch (java.security.NoSuchAlgorithmException e) {
-			return e.getMessage();
-		}
-	}
-
-	private String certChainMessage(final X509Certificate[] chain, CertificateException cause) {
-		Throwable e = cause;
-		Log.d(TAG, "certChainMessage for " + e);
-		StringBuffer si = new StringBuffer();
-		if (e.getCause() != null) {
-			e = e.getCause();
-			si.append(e.getLocalizedMessage());
-			//si.append("\n");
-		}
-		for (X509Certificate c : chain) {
-			si.append("\n\n");
-			si.append(c.getSubjectDN().toString());
-			si.append("\nMD5: ");
-			si.append(certHash(c, "MD5"));
-			si.append("\nSHA1: ");
-			si.append(certHash(c, "SHA-1"));
-			si.append("\nSigned by: ");
-			si.append(c.getIssuerDN().toString());
-		}
-		return si.toString();
-	}
-
-	void startActivityNotification(Intent intent, String certName) {
```

Comment:
Gotcha, in the future if a class needs formatting changes, try to do a separate PR with just those changes.